db-service/lib/cqn2sql.js

```diff
@@ -34,6 +34,12 @@ class CQN2SQLRenderer {
     this.class = new.target // for IntelliSense
     this.class._init() // is a noop for subsequent calls
     this.model = srv?.model
+
+    // Overwrite smart quoting
+    if (cds.env.cdsc.sqlMapping === 'quoted') {
+      this.class.prototype.name = (name) => name.id || name
+      this.class.prototype.quote = (s) => `"${s.replace(/"/g, '""')}"`
+    }
```

Comment:
This is the setting which lets the compiler use quoted mode as well. Would it be useful to have a runtime independent setting for quotation?

Reply:
The Database Service has to follow the quoting behaviors of the compiler. So from my perspective it only makes sense to have them both react to the exact same configuration.

Reply:
👍